eogrow/utils/types.py

```diff
@@ -14,3 +14,5 @@
 FeatureSpec = Union[Tuple[FeatureType, str], FeatureType]
 
 BoolOrAuto = Union[Literal["auto"], bool]
+
+ExecutionKind = Literal["single", "multi", "ray"]
```

Comment:
I would definitely make this an `Enum` so that we don't have to work with strings in the code. I see that we already have `eolearn.core.utils.parallelize._ProcessingType` which is basically what we want here. But I'm not sure if we should make that enum public or recreate it here :thinking:.

Reply:
I don't mind keeping it as a string since we're mypy-checked, but I can also switch it to Enum.

Reply:
The only extra benefit of having an enum over literal is that you get auto-complete when you write the code. E.g. when I write `if execution_kind == "ray":` I don't get an auto-complete on `"ray"` in PyCharm.

Reply:
Since we're using enums extensively throughout our codebase I think it is the correct thing to do so here as well. Will adjust.

Reply:
done